plugins/terraform/src/components/TerraformLatestRun/utils.ts

```diff
@@ -0,0 +1,8 @@
+import { Entity } from "@backstage/catalog-model";
+import { TERRAFORM_WORKSPACE_ANNOTATION, TERRAFORM_WORKSPACE_ORGANIZATION } from "../../annotations";
+
+export const getAnnotations = (entity: Entity): { organization?: string, workspace?: string } => {
```

Comment:
Nice, could you move this to a common util and then update the Terraform component to also use this?